pkg/oc/admin/router/router.go

```diff
@@ -239,6 +244,12 @@ const (
 
 	// Default stats port.
 	defaultStatsPort = 1936
+
+	rsyslogConfigurationFile = `$ModLoad imuxsock
```

Comment:
This is the very old rsyslog configuration syntax.  Why are we not using the new syntax?

Reply:
I used the syntax described in the manpage.  What is the new syntax?

(Note that this code has since all been deleted, but it would be useful to know how it should be done in case we introduce similar code in the ingress operator.)

Reply:
```
module(load="imuxsock")
module(load="omstdout")
input(type="imuxsock" Socket="/var/lib/rsyslog/rsyslog.sock" SysSock.RateLimit.Interval="0")
*.* action(type="omstdout")
```

Reply:
Thanks!  Does the fact that the manpage and standard configuration files have the old syntax (or at least the fact that the manpage doesn't mention the new syntax) warrant a Bugzilla report?

Reply:
Using the configuration that you provided with the new syntax caused the following errors:
```
rsyslogd: error during parsing file /etc/rsyslog/rsyslog.conf, on or before line 3: parameter 'SysSock.RateLimit.Interval' not known -- typo in config file? [v8.24.0-34.el7 try http://www.rsyslog.com/e/2207 ]
rsyslogd: config objects are not supported by module 'omstdout' -- legacy config options MUST be used instead [v8.24.0-34.el7 try http://www.rsyslog.com/e/2210 ]
rsyslogd: error during parsing file /etc/rsyslog/rsyslog.conf, on or before line 4: errors occured in file '/etc/rsyslog/rsyslog.conf' around line 4 [v8.24.0-34.el7 try http://www.rsyslog.com/e/2207 ]
rsyslogd: CONFIG ERROR: there are no active actions configured. Inputs will run, but no output whatsoever is created. [v8.24.0-34.el7 try http://www.rsyslog.com/e/2103 ]
rsyslogd: cannot create '/dev/log': Permission denied [v8.24.0-34.el7]
```
I can get past the `parameter 'SysSock.RateLimit.Interval' not known` and `cannot create '/dev/log'` errors with the following configuration:
```
module(load="imuxsock" SysSock.Use="off")
module(load="omstdout")
input(type="imuxsock" Socket="/var/lib/rsyslog/rsyslog.sock" RateLimit.Interval="0")
*.* action(type="omstdout")
```
However, I still get the `config objects are not supported by module 'omstdout'`.  It does not appear to be possible to use the omstdout module with the new syntax.